src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const ACTIVE = 'ACTIVE';
+const NOT_ACTIVE = 'NOT_ACTIVE';
```

Comment:
- [ ] At this point, you need only one action type, not two. Kindly remove the second action type to avoid redundancy.

- [ ] **[OPTIONAL]** I would recommend you name your action type something more meaningful, like CHECK_STATUS, since that best expresses what we are trying to achieve.

- [ ] Kindly ensure you follow the ducks pattern for your values, like so: `bookstore/categories/CHECK_STATUS`